appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/connectors/service/ConnectorResourceAdminServiceImpl.java

```diff
@@ -66,15 +66,15 @@ public void createConnectorResource(ResourceInfo resourceInfo, PoolInfo poolInfo
         try {
             ccp = (ConnectorConnectionPool) namingService.lookup(poolInfo, jndiNameForPool);
         } catch (NamingException ne) {
-            _logger.log(Level.WARNING,
+            _logger.log(Level.INFO,
                 "Probably the pool {0} is not yet initialized (lazy-loading), trying to check ...", poolInfo);
             try {
                 checkAndLoadPool(poolInfo);
                 ccp = (ConnectorConnectionPool) namingService.lookup(poolInfo, jndiNameForPool);
             } catch (NamingException e) {
-                _logger.log(Level.SEVERE, "Second lookup failed for {0}: {1}", new Object[] {poolInfo, e});
```

Comment:
Just a question - isn't the error logged twice? I am usually deleting these logs like this if the exception is thrown immediately after it. 
The only exception is when the thrown exception is somehow processed and then we would not have the stacktrace in logs (also this log doesn't produce stacktrace, just `poolInfo.toString` and `e.toString`) or if it is logged on lower level.

Reply:
As you said, the error logs are duplicated because the exception message is also printed in:  

https://github.com/eclipse-ee4j/glassfish/blob/7bfd4527f92e1f76115af814c630da880a6c7bc0/nucleus/resources/src/main/java/org/glassfish/resourcebase/resources/listener/ResourceManager.java#L430

I've corrected it.